docs/server/auth.md

```diff
@@ -260,6 +260,9 @@ With this set, now you can continue your [Using Spark SQL to query Unity Catalog
 
 We previously configured Google as the Identity Provider and configured UC Server settings for CLI access. However, we can also apply this authentication and authorization to the Unity Catalog UI. 
 
+!!! note "First time using Unity Catalog UI?"
+     If this is your first time interacting with Unity Catalog UI, you might want to check out the [Unity Catalog UI](./ui.md) first.
```

Comment:
There is no ./ui.md page per the lint test - could you please update to the valid location ?

Reply:
Please refer to [this check](https://github.com/unitycatalog/unitycatalog/actions/runs/11305261771/job/31444571561?pr=597) - thanks!

Reply:
Thank you for addressing these issues. I have corrected them and committed the changes. - thanks!